pkgs/tools/typesetting/tex/texlive/bin.nix

```diff
@@ -24,15 +24,15 @@ let
       # FIXME: a proper mirror, though tarballs.nixos.org saves this case ATM
```

Comment:
IIRC tarballs.nixos.org does not add everything automatically to the content-address source store; a maintainer has to run this:
https://github.com/NixOS/nixpkgs/blob/master/maintainers/scripts/copy-tarballs.pl

Finding broken/moved tarballs should be easier now, because we're using upstream's `sha512` hashes as published on CTAN instead of adding the layer of indirection to download -> verify -> re-hash with `sha1`; but nevertheless it'd be nice to have the ability to fetch the missing source tarballs from Hydra!

Reply:
IIRC the script doesn't work on texlive, because we don't use pure fetching (we unpack, modify a bit, and fix hash of the result).

Reply:
The sources for the binary in this file (`bin.nix`) should be ok, they are archived by release date and should be stable enough. 

The issue is with the thousands of package source tarballs (see `default.nix`), which are not under version control. For packages, the upstream package database file and our generated `pkgs.nix` now both use unmodified `sha512` hashes of the downloaded `.tar.xz` files - so maybe uploading them to `tarballs.nixos.org` could work now?